templates/store/snap-details/_distro-instructions-for-snap-support.html

```diff
@@ -0,0 +1,109 @@
+<div>
+    <div class="row">
+        <div class="col-12">
+            <hr />
+        </div>
+    </div>
+</div>
+<div class="p-strip is-shallow">
+    <div class="row">
+        <div class="col-12">
+        <h3 class="p-heading--two">Install {{snap_title}} on your Linux distribution</h3>
+        <p>Choose your Linux distribution to get detailed installation instructions. If yours is not shown, get more details on the <a href="https://docs.snapcraft.io/installing-snapd" target="_blank">installing snapd documentation</a>.</p>
+        </div>
+    </div>
+</div>
+<div class="p-strip is-shallow u-no-padding--top">
+    <div class="row" id="distros">
+        <div class="col-12 snapcraft-p-grid">
+            <a class="p-media-object--small" href="/install/{{package_name}}/arch">
+            <span class="p-media-object__image">
+                <img class="p-media-object__image" src="https://assets.ubuntu.com/v1/feca0fc0-Distro_Logo_ArchLinux.svg" alt="">
```

Comment:
`p-media-object__image` class is duplicated on span and img. I guess span is not necessary at all?